include/mbedtls/config_psa.h

```diff
@@ -1003,13 +1003,26 @@ extern "C" {
 #define PSA_WANT_ALG_SOME_PAKE 1
 #endif
 
-/* Temporary internal migration helpers */
-#if defined(PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_BASIC) || \
-    defined(PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_IMPORT) || \
-    defined(PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_EXPORT) || \
-    defined(PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_GENERATE) || \
-    defined(PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_DERIVE)
-#define MBEDTLS_PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_LEGACY
+/* Even though KEY_PAIR symbols' feature several level of support (BASIC, IMPORT,
+ * EXPORT, GENERATE, DERIVE) we're not planning to have support only for BASIC
+ * without IMPORT/EXPORT since these last 2 features are strongly used in tests.
+ * This is possible also because, in general, it is allowed to include more
+ * feature than what is strictly requested.
+ *
+ * As a consequence the following internal symbol:
+ * - is used to put together these 3 dependencies (BASIC + IMPORT + EXPORT)
+ * - is enabled by BASIC which itself is enabled as soon as any feature is
+ *   selected
+ * - it does not include EXPORT or GENERATE, for which specific symbols have
+ *   to be enabled explicitly.
+ */
+#if defined(PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_BASIC)
+#define MBEDTLS_PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_BASIC_IMPORT_EXPORT
```

Comment:
I don't understand why this requires yet another symbol. Why not simply
```
if defined(PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_BASIC)
#define PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_IMPORT
#define PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_EXPORT
#endif
```

Reply:
Fair question. The reason I went for yet another symbol is so that if at some point in the future we want to support builds without import or export, the extra symbol should make it clearer to distinguish between "this is really about basic" vs "this has not been reviewed / classified / replaced yet" while we work on that.

If you think it's not worth it, I won't insist on it.